### ISSUE TITLE
[BUG] hover tooltip stretches badly to the left

### ISSUE THREAD

**Issue by bryevdv** [bokeh/bokeh#9717]
Tooltips going to the left anchor themselves to the left edge and stretch to the cursor instead of always displaying a tight fitting box:

![ScreenFlow](https://user-images.githubusercontent.com/1078448/75132665-b3318a80-568c-11ea-8b5b-bdb49b61d1c9.gif)

@jsignell do you have any quick suggestions?

**Comment by bryevdv** [comment]
Quick observation is that disabling `left: inherit` fixes:
<img width="568" alt="Screen Shot 2020-02-23 at 10 39 22 PM" src="https://user-images.githubusercontent.com/1078448/75132861-65695200-568d-11ea-9e01-3e87ebdee80b.png">

However:

* right facing tooltips have `right: inherit` and don't suffer this problem
* not sure if removing that would cause other problems

**Comment by bryevdv** [comment]
This is pretty glaring, I'd like to fix for 2.0

**Comment by bryevdv** [comment]
This diff seems to fix:
```diff --git a/bokehjs/src/lib/models/annotations/tooltip.ts b/bokehjs/src/lib/models/annotations/tooltip.ts
index a96499645..18a426984 100644
--- a/bokehjs/src/lib/models/annotations/tooltip.ts
+++ b/bokehjs/src/lib/models/annotations/tooltip.ts
@@ -115,8 +115,8 @@ export class TooltipView extends AnnotationView {
     // be problematic
     if (this.el.childNodes.length > 0) {
       this.el.style.top = `${top}px`
-      this.el.style.left = left ? `${left}px` : 'inherit'
-      this.el.style.right = right ? `${right}px` : 'inherit'
+      this.el.style.left = left ? `${left}px` : 'auto'
+      this.el.style.right = right ? `${right}px` : 'auto'
     } else
       undisplay(this.el)
   }
```

setting to `initial` instead of `auto` also seems to work. This is just some trial and error though.
